File: src/com/xter/callback/Gesture.java
package com.xter.callback;

/**
 * @author XTER
 * 项目名称: TechBasis
 * 创建时间: 2020/5/8
 * 描述:
 */
public interface Gesture {
	void onMove();

	void setGestureListener(GestureListener<? extends Gesture> listener);
}

File: src/com/xter/callback/kotlin/Gesture.kt
package com.xter.callback.kotlin

/**
 * @author XTER
 * 项目名称: TechBasis
 * 创建时间: 2020/5/8
 * 描述:
 */
interface Gesture {
    fun onMove()

    fun setGestureListener(listener: GestureListener<out Gesture>)
}
